onnx/defs/gen_doc.py

```diff
@@ -214,6 +214,14 @@ def support_level_str(level):  # type: (OpSchema.SupportType) -> Text
         "<sub>experimental</sub> " if level == OpSchema.SupportType.EXPERIMENTAL else ""
 
 
+class Schema(object):
```

Comment:
Can you add a brief note why the changes in this file are needed? I vaguely remember seeing a thread (you may have written) long back about some motivating problem. It would be good to document it. Thanks!

Reply:
Sorry, I didn't intent this tp be a part of this PR.